src/databricks/labs/ucx/cli.py

```diff
@@ -113,6 +114,44 @@ def ensure_assessment_run():
         workspace_installer.validate_and_run("assessment")
 
 
+def revert_migrated_tables(schema: str, table: str, *, deletemanaged: bool = False):
+    ws = WorkspaceClient()
+    prompts = Prompts()
+    installation_manager = InstallationManager(ws)
+    installation = installation_manager.for_user(ws.current_user.me())
+    if not schema and not table:
+        if not prompts.confirm(
+            "You haven't specified a schema or a table. All migrated tables will be reverted."
+            " Would you like to continue?",
+            max_attempts=2,
+        ):
+            return None
+    if not installation:
+        logger.error(CANT_FIND_UCX_MSG)
+        return None
+    warehouse_id = installation.config.warehouse_id
+    sql_backend = StatementExecutionBackend(ws, warehouse_id)
+    table_crawler = TablesCrawler(sql_backend, installation.config.inventory_database)
+    tm = TablesMigrate(table_crawler, ws, sql_backend)
+    migrated_count = tm.get_migrated_count()
```

Comment:
Can you move this check to a TablesMigrate class? cli.py must not grow big :)

Reply:
What exactly am I moving?

Reply:
Fixed